### PR TITLE
Edit to Get DEE CentOS topic in line with DEE Ubuntu.

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**
As with the GET DEE for Ubuntu topic, there is a Docker Hub reference here that needs to be reconciled.

### CODE REVIEW

File: ee/docker-ee/centos.md [hunk 1]
```diff
@@ -64,28 +62,24 @@ apply:
...
-### Uninstall old Docker versions
+### Uninstalling All Old Docker Versions
 
-The Docker Engine - Enterprise package is called `docker-ee`. Older versions
-were called `docker` or `docker-engine`. Uninstall all older versions and
-associated dependencies. The contents of `/var/lib/docker/` are preserved,
-including images, containers, volumes, and networks. If you are upgrading from
-Docker Engine - Community to Docker Engine - Enterprise, remove the Docker
-Engine - Community package as well.
+Use the `yum remove` command to uninstall older versions and associated dependencies of Docker Engine - Enterprise (called `docker` or `docker-engine`. Note that the contents of `/var/lib/docker/` are preserved, including images, containers, volumes, and networks. In addition, if you are upgrading from Docker Engine - Community to Docker Engine - Enterprise, remove the Docker Engine - Community package.
```
Comment (traci-morrison) [inline ee/docker-ee/centos.md]: Add ")" after `docker-engine`

File: ee/docker-ee/centos.md [hunk 2]
```diff
@@ -159,74 +154,76 @@ Shared between centOS.md, oracle.md
...
 
-2.  To install a _specific version_ of Docker Engine - Enterprise (recommended in production), list versions and install:
+    **— or —**
 
-    a.  List and sort the versions available in your repo. This example sorts results by version number, highest to lowest, and is truncated:
+    * To install a _specific version_ of Docker Engine - Enterprise (recommended in production), list versions and install:
 
-    ```bash
-    $ sudo yum list docker-ee  --showduplicates | sort -r
+        a. List and sort the versions available in your repo. This example sorts   results by version number, highest to lowest, and is truncated:
```
Comment (traci-morrison) [inline ee/docker-ee/centos.md]: Extra space between "sorts" and "results"

File: ee/docker-ee/centos.md (hunk 1 above)
Comment (Dawn-Wood) [reply ee/docker-ee/centos.md]: You could also remove the "Note that" and start the sentence with "The contents of ..."

File: ee/docker-ee/centos.md [hunk 3]
```diff
@@ -64,28 +62,24 @@ apply:
   solid-state media (SSD) is recommended. Do not start Docker until properly
   configured per the [storage guide](/storage/storagedriver/device-mapper-driver/){: target="_blank" class="_" }.
 
-### Find your Docker Engine - Enterprise repo URL
+### Locating the URL for the Docker Engine - Enterprise Repo
 
-To install Docker Enterprise, you will need the URL of the Docker Enterprise repository associated with your trial or subscription:
+The product URL associated with your trial or subscription is required to install Docker Engine - Enterprise, which can be attained using the following procedure (note that these instructions apply to Docker on CentOS and for Docker on Linux, which includes access to Docker Engine - Enterprise for all Linux distributions):
```
Comment (Dawn-Wood) [inline ee/docker-ee/centos.md]: Could you say "...install Docker Engine - Enterprise. You can use the following procedure to get the required url. These instructions apply to ..."

File: ee/docker-ee/centos.md [hunk 4]
```diff
@@ -100,54 +94,55 @@ $ sudo yum remove docker \
                   docker-engine
 ```
 
-## Repo install and upgrade
+## Install Docker Engine - Enterprise
+
+Docker Engine - Enterprise can be installed either via YUM repositories, or by downloading and installing the RPM package and thereafter manually managing all upgrades. The Docker repository method is recommended, for the ease it lends in terms of both installation and upgrade tasks. The more manual RPM package approach, however, is useful in certain situations, such as installing Docker on air-gapped system that have no access to the Internet.
```
Comment (Dawn-Wood) [inline ee/docker-ee/centos.md]: don't think you need a comma after "recommended". Also wonder if you need "thereafter".

File: ee/docker-ee/centos.md [hunk 5]
```diff
@@ -100,54 +94,55 @@ $ sudo yum remove docker \
...
-
-1.  Remove existing Docker repositories from `/etc/yum.repos.d/`:
+1.  Remove existing Docker repositories from `/etc/yum.repos.d/`.
 
     ```bash
     $ sudo rm /etc/yum.repos.d/docker*.repo
     ```
 
-2.  Temporarily store the URL (that you [copied above](#find-your-docker-ee-repo-url)) in an environment variable. Replace `<DOCKER-EE-URL>` with your URL in the following command. This variable assignment does not persist when the session ends:
+2. Temporarily add a `$DOCKER_EE_URL` variable into your environment (it persists only up until you log out of the session). Replace `<DOCKER-EE-URL>` with the URL you noted down in the [prerequisites](#prerequisites).
```
Comment (Dawn-Wood) [inline ee/docker-ee/centos.md]: is "into" or "in" the right word here?

File: ee/docker-ee/centos.md [hunk 6]
```diff
@@ -236,59 +233,59 @@ Shared between centOS.md, rhel.md, oracle.md
 Shared between centOS.md, rhel.md, oracle.md
 --->
 
-## Package install and upgrade
+### Installing and Upgrading from an RPM Package
 
-To manually install Docker Enterprise, download the `.{{ package-format | downcase }}` file for your release. You need to download a new file each time you want to upgrade Docker Enterprise.
+To manually install Docker Enterprise, download the `.{{ package-format | downcase }}` file for your release. Note that it will be necessry to download a new file each time you want to upgrade Docker Enterprise.
```
Comment (Dawn-Wood) [inline ee/docker-ee/centos.md]: do you need the "Note that"?

**Comment by usha-mandya** [comment]
Closing this PR as it relates to Enterprise docs which are no longer maintained in the Docker Git repository.

**Pull request closed without merging**